Makefile

```diff
@@ -428,3 +428,6 @@ operator-sdk: $(OPERATOR_SDK)  ## Download operator-sdk locally if necessary
 $(OPERATOR_SDK):
 	curl --silent --show-error --location --fail "https://github.com/operator-framework/operator-sdk/releases/download/v1.10.1/operator-sdk_linux_amd64" --output $(OPERATOR_SDK)
 	chmod +x $(OPERATOR_SDK)
+
+scorecard-tests: bundle ## Run the scorecard tests
```

Comment:
Can we call the new target `run-scorecard-tests`?  This will keep it common with the other test targets (run-unit-tests, run-int-tests, etc.).